repository/src/main/java/ru/karalionak/rest/query/SortingContext.java

```diff
@@ -0,0 +1,21 @@
+package ru.karalionak.rest.query;
+
+import java.util.List;
+
+public class SortingContext {
+    private final List<String> columns;
```

Comment:
It's not safety to retrieve column names exctaly from user. Use enum to avoid SQL syntax errors.